### PR TITLE
niti: teach the interpreter to read from the standard input

### PR DESCRIPTION
**Pull request by privat**
With `-` the program is read from stdin.

~~~
$ echo H4sIAN7lUFcAAysoyswrUVDySM3JyVfiAgCjfiaWDgAAAA== | base64 -d | gunzip | nit -
Hello
~~~

### CODE REVIEW

**Comment by xymus** [comment]
+1

File: share/man/nit.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ nit - interprets and debugs Nit programs.
 
 nit [*options*] FILE [ARG]...
 
+nit [*options*] - [ARG]...
```
Comment (jcbrinfo) [inline share/man/nit.md]: Using `-` in place of a path is standard. I should not add a synopsis line for that.

Comment (jcbrinfo) [inline share/man/nit.md]: See [POSIX XCU, section 12.2](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_02), guideline 13.

**Comment by jcbrinfo** [comment]
:+1: +1

**Pull request merged**